lang/fpc/Portfile

```
@@ -220,12 +220,15 @@ subport "${name}-cross-x86-64-win64" {
     }
 }
 
-if {${subport} eq "${name}"} {
+if {${subport} == "${name}"} {
```

I think `eq` is in fact correct.

---

I had a mix of eq and == and wanted to have one only.

---

`==` is used for numerical values, `eq` for strings.

---

oh. I see.

---

So can we fix this please?
```suggestion
if {${subport} eq "${name}"} {
```

---

done